frontend/src/console/components/RedeploymentToolbar/styles.scss

```diff
@@ -57,3 +56,15 @@
     margin-bottom: 15px;
   }
 }
+
+.alert-info {
+  width: 90%;
+  font-family: "Roboto", sans-serif;
+  background-color: #6d9cae;
+  align-items: center;
+  display: flex;
+  margin: 0rem;
+  justify-content: center;
+  border-radius: 15px;
+  padding: 0.5rem;
+}
```

Comment:
Let's keep custom styling to a minimum, keep whatever we get from bootstrap and add only the bare minimum needed. 

Also, don't directly use font names or colour values. Instead, use the defined variables for our theme colours. Here just use the default colours for alert. The theme should set them, you shouldn't override them.

Reply:
I have changed it to a component :+1: